### DISCUSSION TITLE
Unsupported Chain

### DISCUSSION THREAD

**Discussion by LilFatFrank** [family/connectkit#178] [Q&A] [answered]
Using connectkit with wagmi
When you switch to a chain that is not supported, it returns an error like so

![image](https://user-images.githubusercontent.com/30598417/228642435-9800ea58-9cb7-4a3c-9acc-3a45cf57c405.png)

Is there a way to remove this modal?

**Comment by lochie** [comment, marked as answer]
@LilFatFrank you can stop this by disabling `enforceSupportedChains`

See docs for more:
https://docs.family.co/connectkit/api-reference#ConnectKitProvider-enforceSupportedChains
